crates/block-explorers/src/lib.rs

```diff
@@ -671,20 +564,9 @@ mod tests {
     }
 
     #[test]
-    fn test_api_paths_v1() {
-        let client =
-            Client::new_with_api_version(Chain::goerli(), "", EtherscanApiVersion::V1).unwrap();
-        assert_eq!(client.etherscan_api_url.as_str(), "https://api-goerli.etherscan.io/api");
-
-        assert_eq!(client.block_url(100), "https://goerli.etherscan.io/block/100");
-    }
-
-    #[test]
-    fn test_api_paths_v2() {
-        let client =
-            Client::new_with_api_version(Chain::goerli(), "", EtherscanApiVersion::V2).unwrap();
+    fn test_api_paths() {
+        let client = Client::new(Chain::goerli(), "").unwrap();
```

Comment:
I know this is not from this PR, so somewhat unrelated, but it feels weird having empty strings here